### DISCUSSION TITLE
What's the difference between tag and version?

### DISCUSSION THREAD

**Discussion by healthjyk** [goreleaser/goreleaser#4243] [Q&A] [answered]
GoReleaser provides `{{ .Version }}`, which is the `{{ .Tag }}` that removes prefix `v`. Why cannot the version and tag be the same? It makes me really confused.
For example, I find that for the docker image, the tag with prefix v is used shown in the recommended example, while for the homebrew, the version without prefix v is used. It seems that tag with prefix v is because of golang, but why should it get trimmed, keep the same seems much simpler!!!

**Comment by caarlos0** [comment, marked as answer]
For the simple case it might look like one is just the other without the preceding `v`, but it is a bit more complex than that:

The Version can be changed by the snapshot and nightly configurations, whereas the tag is static.

Also, if your tag is something like `foo/v1.2.3`, with the monorepo configurations, you can make Version be `1.2.3`, which is what you'll likely want to use most places.

Hopefully that clarifies it a bit.
